skrf/network.py

```diff
@@ -2717,6 +2718,14 @@ def extrapolate_to_dc(self, points: int = None, dc_sparam: NumberLike = None, ki
         impulse_response
         step_response
         """
+        if kind == 'warn':
```

Comment:
Why did you introduce `kind='warn'`? Can't you just keep `kind='rational'` and check against this value to print the warning? For example:
```
if kind == 'rational':
    warnings.warn('...')
```

Reply:
The warning would be printed every time, even if you select `kind='rational'` excplicit.

Reply:
Oh, you're right.